tests/regression_tests/deplete_with_transfer_rates/test.py

```diff
@@ -0,0 +1,82 @@
+""" TransferRates depletion test suite """
+
+from pathlib import Path
+
+import numpy as np
+import pytest
+import openmc
+import openmc.deplete
+from openmc.deplete import CoupledOperator
+from openmc.deplete import TransferRates
+
+@pytest.fixture
+def model():
+    f = openmc.Material(name="f")
+    f.add_element("U", 1, percent_type="ao", enrichment=4.25)
+    f.add_element("O", 2)
+    f.set_density("g/cc", 10.4)
+
+    w = openmc.Material(name="w")
+    w.add_element("O", 1)
+    w.add_element("H", 2)
+    w.set_density("g/cc", 1.0)
+    w.depletable = True
+
+    radii = [0.42, 0.45]
+    f.volume = np.pi * radii[0] ** 2
+    w.volume = np.pi * (radii[1]**2 - radii[0]**2)
+
+    materials = openmc.Materials([f, w])
+
+    surf_f = openmc.Sphere(r=radii[0])
+    surf_w = openmc.Sphere(r=radii[1], boundary_type='reflective')
+    cell_f = openmc.Cell(fill=f, region=-surf_f)
+    cell_w = openmc.Cell(fill=w, region=+surf_f & -surf_w)
+    geometry = openmc.Geometry([cell_f, cell_w])
+
+    settings = openmc.Settings()
+    settings.particles = 1000
```

Comment:
The tests below with power=174 each take about 12 seconds on my system and will probably be slower on CI. Let's cut down the particles here even further to 100 which should really minimize the time.
```suggestion
    settings.particles = 100
```